applications/opentelemetry-operator/0.93.0/helmrelease/opentelemetry.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: opentelemetry-operator
```

Comment:
```suggestion
  name: opentelemetry-0.93.0-operator
```
fyi starting 2.17 we can use `${releaseName}-${appVersion}-operator` but since this is targetted for 2.16 and 2.17 we can't do that here.